corehq/apps/app_manager/views/forms.py

```diff
@@ -722,7 +726,9 @@ def commtrack_programs():
             "app_manager/v1/form_view.html",
             "app_manager/v2/form_view.html",
         )
-        return template, context
+
+    context.update({'case_config_options': case_config_options})
```

Comment:
In future PRs, there's going to be more added to that dict in the module-type-based conditionals above this.

> Also, this function looks like a great candidate for the next refactorathon
yeah, tell me about it...